go.mod

```diff
@@ -8,6 +8,7 @@ require (
 	github.com/google/shlex v0.0.0-20191202100458-e7afc7fbc510
 	github.com/sonatype-nexus-community/gonexus v0.59.0
 	github.com/tektoncd/pipeline v0.24.0
+	golang.org/x/net v0.0.0-20210224082022-3d97a244fca7
```

Comment:
@gerardcl Hey I might be too late, but is there a reason this is a relatively old version?
See https://pkg.go.dev/golang.org/x/net@v0.0.0-20220127200216-cd36cc0744dd

Reply:
this was the version installed automatically when running the tidy command, and we run that command like a week ago, so I think it is coincidence there is a recent new version 👍

Reply:
I think this is a byproduct of the minimal version selection algorithm applied by the Go module system. `go.sum` already contains a `golang.org/x/net` reference, so some other module is already depending on that package. Your changes only triggered `golang.org/x/net` to be in `go.mod` now, because you are directly depending on it now. `go mod tidy` shouldn't change the version I believe either. You can raise the version in `go.mod` though by doing something like `go get golang.org/x/net@master` I believe.